packages/website/src/components/ErrorsViewer.tsx

```diff
@@ -1,11 +1,13 @@
-import React, { useEffect, useMemo, useState } from 'react';
+import React, { useEffect, useState } from 'react';
 import type Monaco from 'monaco-editor';
-import type { ErrorItem } from './types';
+import clsx from 'clsx';
 
+import type { ErrorItem, ErrorGroup } from './types';
+import LinkIcon from '@site/src/icons/link.svg';
```

Comment:
Hmm, does Docusaurus not export a way to use their svg used for external header links?

Reply:
Yes, there's `@theme/IconExternalLink`.